perl-Bootloader-testsuite/tests/test_interface/backup/fake_root1/boot/extlinux/extlinux.conf

```diff
@@ -0,0 +1,25 @@
+TIMEOUT 1000
+ONTIMEOUT localboot
+DEFAULT default
+MENU TITLE Boot menu
+PROMPT 0
+
+LABEL default
+         MENU LABEL Default
+         LINUX /boot/zImage
+         FDTDIR /boot/dtb/
+         FDT exynos5250-snow.dtb
```

Comment:
While I agree that FDT should not unnecessarily be specified, does it do any harm in the testsuite here? If I understood perl-bootloader correctly, it needs to parse and retain anything a user put into the file, so having a test case covering everything seems good.